internal/file/zip_file_traversal.go

```diff
@@ -91,6 +92,12 @@ func ExtractFromZipToUniqueTempFile(archivePath, dir string, paths ...string) (m
 		// provides a ReadCloser. It is up to the caller to handle closing the file explicitly.
 		defer tempFile.Close()
 
+		// grab and assign digest for the visited zip file
+		digest, err := Digest(file.FileInfo().Name())
+		if err != nil {
+			log.Warnf("failed to parse digest for file (%s): %+v", file.Name, err)
```

Comment:
Re: this message... Are we "parsing" the digest? IIUC we're just calculating it

Reply:
calculating, I'll update the message here